NOTES-NONSTOP.md

```diff
@@ -78,12 +78,64 @@ The current OpenSSL default memory model uses the default platform address
 model. If you need a different address model, you must specify the appropriate
 c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
 
+Cross Compiling on Windows
+--------------------------
+
+To configure and compile OpenSSL, you will need to set up a Cygwin environment.
+The Cygwin tools should include bash, make, and any other normal tools required
+for building programs.
+
+The `PATH` must include the bin directory for the c99 cross-compiler, as in:
+
+    export PATH=/cygdrive/c/Program\ Files\ \(x86\)/HPE\ NonStop/L16.05/usr/bin:$PATH
+
+This should be set before Configure is run. For the c99 cross-compiler to work
+correctly, you also need the COMP_ROOT set, as in:
+
+    export COMP_ROOT="C:\Program Files (x86)\HPE NonStop\L16.05"
+
+`COMP_ROOT` needs to be in Windows form.
+
+`Configure` must specify the `no-makedepend` option otherwise errors will
+result when running the build because the c99 cross-compiler does not support
+the `gcc -MT` option. An example configure command is run from the OpenSSL
```

Comment:
The wording _An example configure command is run from the OpenSSL directory as follows:_ reads oddly to me.

How about: _An example of a configure command to be run from the OpenSSL directory is:_